### ISSUE TITLE
no way to compile twitcher

### ISSUE THREAD

**Issue by belldandu** [scuq/agtv#32]
How are we supposed to compile twitcher?

**Comment by scuq** [comment]
well' it should compile by just using the .pro file with qt-creator, but I haven't touched this branch for quite some time, if you give me a few days I'll look into it, because it should be merged to agtv anyhow.

**Comment by scuq** [comment]
we are talking about the twitcher branch aren't we?

**Comment by belldandu** [comment]
Yeah also i saw no way to edit twitch stream info in agtv only settings for
joining chat and viewing video

**Comment by scuq** [comment]
you mean you'd like to stream and set your stream infos? yes that was a twitcher feature, the "viewer" part is agtv, I'd like to join the code since there were some major changes in twitchapi within agtv, weekend is coming, so sometime between now and sunday I'll look into it :)

**Comment by scuq** [comment]
just some delay here, started reintegrating twitcher today

**Comment by belldandu** [comment]
awsome
